sw/device/silicon_creator/lib/base/sec_mmio.c

```diff
@@ -7,8 +7,15 @@
 #include "sw/device/lib/base/memory.h"
 #include "sw/device/silicon_creator/lib/base/abs_mmio.h"
 
-// FIXME: Linker configuration.
+#if HOST_BUILD
+// Tests running on the host define their own context.
 extern sec_mmio_ctx_t sec_mmio_ctx;
+#else
+// On device the context is located in the .critical_static section at the
+// beginning of main RAM.
+__attribute__((section(
```

Comment:
Nit: you can get better wrapping behavior by writing
```c
__attribute__((...)) //
volatile sec_mmio_ctx_t sec_mmio_ctx;
```

Reply:
Thanks, done.